oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/banyandb/client/request/TraceSearchQuery.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.banyandb.client.request;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import org.apache.skywalking.banyandb.Query;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class TraceSearchQuery {
+    private final String key;
+    private final TraceSearchRequest.BinaryOperator op;
+    private final Object value;
+    private final Query.TypedPair.TypedCase typedCase;
+
+    public Query.PairQuery toPairQuery() {
```

Comment:
This seems testing only? Please use `default` scope.

Reply:
No. It is not only used in tests